exporter/metric/metric.go

```diff
@@ -389,7 +393,7 @@ func recordToMdpbKindType(a metricdata.Aggregation) (googlemetricpb.MetricDescri
 			return googlemetricpb.MetricDescriptor_CUMULATIVE, googlemetricpb.MetricDescriptor_DOUBLE
 		}
 		return googlemetricpb.MetricDescriptor_GAUGE, googlemetricpb.MetricDescriptor_DOUBLE
-	case metricdata.Histogram:
+	case metricdata.Histogram[float64]:
```

Comment:
```suggestion
	case metricdata.Histogram[float64], metricdata.Histogram[int64]:
```

Reply:
that was my first thought, unfortunately doesn't work:

`../../metric/metric.go:494:27: a.DataPoints undefined (type "go.opentelemetry.io/otel/sdk/metric/metricdata".Aggregation has no field or method DataPoints)`

which kinda makes sense since `DataPoints []HistogramDataPoint[N]` is also generic

Reply:
going with the ugly dupe

Reply:
resolved the one below to clear it up